packages/ui-components/src/lib/__mocks__/mockTransactionStore.ts

```
@@ -48,22 +47,22 @@ export const mockTransactionStore: Partial<TransactionStore> & {
 	checkingWalletAllowance: (message: string = '') =>
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)